### PR TITLE
Updated bug report template

### PR DESCRIPTION
**Pull request by Trae32566**
This cleans up the bug reporting template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,32 +1,45 @@
...
 labels: ''
 assignees: ''
 
 ---
 
-# Please follow the general troubleshooting steps first:
+Note: If the bug affects multiple games, please open an issue for each game with the game in the title.
 
-- [ ] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and completed successfully the setup wizard.
+# Bug Reports
```
Comment (ckiee) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: No need for a primary header if it shows up in the issue contents

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,32 +1,45 @@
...
+
+## Description
+Please replace this line with a brief summary of your issue
```
Comment (ckiee) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]:
```suggestion
<!-- Please add a brief summary of your issue -->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,32 +1,45 @@
...
+name: Bug Report
 about: Create a report to help us improve
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
-# Please follow the general troubleshooting steps first:
+Note: If the bug affects multiple games, please open an issue for each game with the game in the title.
```
Comment (ckiee) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]:
```suggestion
<!-- Note: If the bug affects multiple games, please open an issue for each game with the game in the title. -->
```

Comment (ckiee) [reply .github/ISSUE_TEMPLATE/bug_report.md:10]: Secondly I'm not sure this policy actually makes sense, if there's an ALVR issue that triggers problems in multiple games then that's just one issue for our POV.

**Review by ckiee — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -1,32 +1,45 @@
...
+- [ ] Packaged (`exe`, `deb`, `rpm`, etc)
+- [ ] Portable (`zip`)
+- [ ] Source
 
-**Please note by far the quickest way to get a new feature is to file a Pull Request.**
+**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions):
 
-We will consider your request but it may be closed if it's something we're not actively planning to work on.
+# Feature Requests
+The quickest way to get a new feature is to file a pull request; these will be considered, but may be closed if they're something we're not actively planning to work on.
```
Comment (ckiee) [inline .github/ISSUE_TEMPLATE/bug_report.md:45]: This also probably doesn't make sense here. This is a form users fill out: have a look at this [example](https://github.com/NixOS/nixpkgs/blob/master/.github/PULL_REQUEST_TEMPLATE.md) ([plaintext](https://raw.githubusercontent.com/NixOS/nixpkgs/master/.github/PULL_REQUEST_TEMPLATE.md) too)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (Trae32566) [reply .github/ISSUE_TEMPLATE/bug_report.md:10]: My primary goal here was fixing formatting, but I do agree that it seems like a silly policy. If you'd like to change it we can, I just assumed there was a reason it was there.

Comment (ckiee) [reply .github/ISSUE_TEMPLATE/bug_report.md:10]: Yeet it away~

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (zarik5) [reply .github/ISSUE_TEMPLATE/bug_report.md:12]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (zarik5) [reply .github/ISSUE_TEMPLATE/bug_report.md:45]:
```suggestion
<!-- Feature Requests
The quickest way to get a new feature is to file a pull request; these will be considered, but may be closed if they're something we're not actively planning to work on. -->
```

**Review by ckiee — APPROVED** [review]
This time I also looked at the rendered version and it looks a bit wack but good enough (:

**Pull request merged**